test/framework/constants.go

```diff
@@ -46,6 +46,31 @@ const (
 	cidrIPv6              = "fd00:fd00::/64"
 )
 
+const (
+	EnvStoreType             = "KUMA_STORE_TYPE"
+	EnvStorePostgresHost     = "KUMA_STORE_POSTGRES_HOST"
+	EnvStorePostgresPort     = "KUMA_STORE_POSTGRES_PORT"
+	EnvStorePostgresUser     = "KUMA_STORE_POSTGRES_USER"
+	EnvStorePostgresPassword = "KUMA_STORE_POSTGRES_PASSWORD"
+	EnvStorePostgresDBName   = "KUMA_STORE_POSTGRES_DB_NAME"
+
+	DefaultPostgresPort     = uint32(5432)
+	DefaultPostgresUser     = "kuma"
+	DefaultPostgresPassword = "kuma"
+	DefaultPostgresDBName   = "kuma"
```

Comment:
I think those env vars belongs to postgres deployment

Reply:
Moved